test/specs/conditions/url_param.yaml

```diff
@@ -0,0 +1,4 @@
+condition:
+  and:
+  - url_param.a>: 5
+  - url_param.a<: 10
```

Comment:
what's the VCL here?

Reply:
Not yet implemented

Reply:
`std.atoi(subfield(req.url.qs, "a", "&"))>5 && std.atoi(subfield(req.url.qs, "a", "&"))<10`. 

https://docs.fastly.com/vcl/functions/std-atoi/
https://docs.fastly.com/vcl/functions/subfield/

might need some protection against strings that can't be read as integers.

Reply:
Done. As it turned out, testing `std.atoi(...)` against a non-number evaluates to `false', no protection required.